grammars/language-idris.cson

```diff
@@ -289,7 +289,7 @@ repository:
         name: 'entity.name.function.idris'
       4:
         name: 'keyword.operator.colon.idris'
-    end: "(;|(?<=[^\\s>])\\s*(?!->)\\s*$)"
+    end: "(;|(?=--)|(?<=[^\\s>])\\s*(?!->)\\s*$)"
```

Comment:
It is a lookahead: http://www.regular-expressions.info/lookaround.html.

Same as here: https://github.com/jackolantern/atom-language-idris/blob/6604e33d3a14bf785ca7774114f3fa03ae3d11f7/grammars/language-idris.cson#L123 and other places throughout the file.